magenta/models/shared/melody_rnn_generate.py

```diff
@@ -80,7 +83,8 @@ def run_generate(graph, train_dir, output_dir, melody_encoder_decoder,
         generated from scratch.
     num_steps: The total number of steps the final melodies should be,
         priming melody + generated steps. Each step is a 16th of a bar.
-    bpm: The tempo in beats per minute that the generated MIDI files will have.
+    temperature: Degree of randomness where default=1
```

Comment:
Actually it looks like temperature is not used in run_generate. You can just take it out of the Args comment.